src/api/services/timetableService.ts

```
@@ -0,0 +1,165 @@
+  content: CourseData[];
+  totalElements: number;
+  totalPages: number;
+  number: number;
+  size: number;
+  first: boolean;
+  last: boolean;
+  empty: boolean;
+}
+
+// Response type for card view (can be different if needed)
+export interface TimetableCardResponse extends TimetableListResponse {}
```

ESLint

[error] An interface declaring no members is equivalent to its supertype.

(@typescript-eslint/no-empty-object-type)